ui/src/app/edge/live/common/grid/modal/modal.html

```diff
@@ -11,15 +11,11 @@
     </oe-modal-line>
 
     <ng-container *ngFor="let meter of meters">
-        <oe-modal-line *ngIf="meters.length > 1" [name]="meter.component.alias ?? meter.component.id"
-            [channelAddress]="meter.component.id + '/ActivePower'" [converter]="Utils.CONVERT_TO_WATT">
+        <oe-modal-line *ngIf="meters.length > 1" [name]="meter.alias ?? meter.id"
+            [channelAddress]="meter.id + '/ActivePower'" [converter]="Utils.CONVERT_TO_WATT">
         </oe-modal-line>
-
-        <!-- If meter is asymmetric, show Phases -->
-        <ng-container *ngIf="meter.isAsymmetric">
-            <oe-modal-meter-phases [component]="meter.component" [setTranslatedName]="setTranslatedName">
-            </oe-modal-meter-phases>
-        </ng-container>
+        <oe-modal-meter-phases [component]="meter" [setTranslatedName]="setTranslatedName">
```

Comment:
Phases are not shown for now.
I am not checking every meter type for now.

This is a UI topic more related to @lukasrgr anyway. 


![grafik](https://user-images.githubusercontent.com/47855186/232490292-923d1e25-2f51-4a1b-8837-ae25b9a8dc9c.png)

Reply:
Problem was existing before already. Fixed in https://github.com/OpenEMS/openems/pull/2144/commits/9f98fc08f2200beb4c9ebbbb50ee7bab7afc10c9